dapps/assets/src/i18n/de.json

```
@@ -30,6 +30,8 @@
       "message": "Nachricht",
       "name": "Name",
       "note": "Notiz",
+      "note-placeholder": "bspw. ein Alias",
+      "pending": "ausstehend ...",
```

```suggestion
      "pending": "ausstehend ...",
```
Lieber konsistent groß schreiben?